sunpy/map/sources/adapt.py

```diff
@@ -0,0 +1,51 @@
+"""
+ADAPT Map subclass definitions
+"""
+
+from astropy.time import Time
+
+from sunpy.map.mapbase import GenericMap, SpatialPair
+
+__all__ = ['ADAPTMap']
+
+class ADAPTMap(GenericMap):
+    """
+    ADAPT magnetic flux map
+
+    The ADAPT (Air Force Data Assimilative Photospheric Flux Transport) flux transport model evolves an ensemble of realizations, using flux transport processes during periods for which there are no observations, and updates the ensemble using the ensemble least-squares (EnLS) data assimilation method to account for model and observational uncertainties.
+    These data products are Carrington maps with an additional third dimension corresponding to the ensemble of models used to produce the ADAPT map.
+
+    The third dimension is ignored and only the first element of the third dimension is used to create the  map.
+
+    .. warning::
+
+        Please note that there are several data arrays within the FITS file and map will try to read them all and fail, so you have to specify the Header Data Pair you want to read.
+        In this case, it will always be the first one.
+        You can do this by passing the ``hdus`` argument to the `sunpy.map.Map` call.
```

Comment:
```suggestion
        Please note that there are several data arrays within the ADAPT FITS files and `~sunpy.map.Map` will, by default, try to read them all and fail.
        In these cases, you must specify the header-data Pair you want to read.
        For these data, it will always be the first one.
        You can specify this by passing the ``hdus`` keyword argument to `~sunpy.map.Map`
```